.venv/Scripts/deactivate.bat

```diff
@@ -1,21 +0,0 @@
-@echo off
```

Comment:
I dont think those script files should be removed.

Reply:
They are untracked, everyone needs to untrack them locally because the scripts on local machine specific. They should never have been in the repo in the first place